integration-tests/README.md

```diff
@@ -0,0 +1,30 @@
+# @eth-optimism/integration-tests
+
+## Setup
+
+Follow installation + build instructions in the [primary README](../README.md).
+Then, run:
+
+```
+yarn build:integration
+```
+
+## Running tests
+
+### Testing a production network
```

Comment:
Nit: The term 'Production' doesn't seem like it would include a Testnet like Kovan. So I find this slightly confusion
 to mean Mainnet. 

Alternatives: 
* Public
* Live
* Staging (excludes mainnet, but a lot of tests get skipped on mainnet anyways).

Reply:
Yeaahhhh I guess. Although I feel like Kovan is sort of a production network since people rely on it actively. Maybe "live" would be best.

Reply:
Changed to "live"